stable/jfrog-platform/values.yaml

```diff
@@ -103,6 +103,10 @@ rabbitmq:
   image:
     repository: bitnami/rabbitmq
     tag: 3.12.10-debian-11-r1
+  podSecurityContext:
```

Comment:
It seems like these changes are specific to you; we are ignoring them

Reply:
I had to add this because otherwise 'helm lint' gives errors about unknown variables.